File: docs/components_page/components/badge/text_color.R
library(dashBootstrapComponents)
library(dashHtmlComponents)

badges <- htmlSpan(
  list(
    dbcBadge("Primary", color = "white", text_color = "primary", className = "border me-1"),
    dbcBadge("Secondary", color = "white", text_color = "secondary", className = "border me-1"),
    dbcBadge("Success", color = "white", text_color = "success", className = "border me-1"),
    dbcBadge("Warning", color = "white", text_color = "warning", className = "border me-1"),
    dbcBadge("Danger", color = "white", text_color = "danger", className = "border me-1"),
    dbcBadge("Info", color = "white", text_color = "info", className = "border me-1"),
    dbcBadge("Dark", color = "white", text_color = "dark", className = "border me-1"),
    dbcBadge("Black", color = "white", text_color = "black", className = "border me-1"),
    dbcBadge("Muted", color = "white", text_color = "muted", className = "border me-1"),
    dbcBadge("Light", color = "secondary", text_color = "light", className = "border me-1"),
    dbcBadge("White", color = "secondary", text_color = "white", className = "border")
  )
)

File: docs/components_page/components/badge/text_color.jl
using DashBootstrapComponents

badges = html_span([
    dbc_badge(
        "Primary",
        color = "white",
        text_color = "primary",
        className = "border me-1",
    ),
    dbc_badge(
        "Secondary",
        color = "white",
        text_color = "secondary",
        className = "border me-1",
    ),
    dbc_badge(
        "Success",
        color = "white",
        text_color = "success",
        className = "border me-1",
    ),
    dbc_badge(
        "Warning",
        color = "white",
        text_color = "warning",
        className = "border me-1",
    ),
    dbc_badge("Danger", color = "white", text_color = "danger", className = "border me-1"),
    dbc_badge("Info", color = "white", text_color = "info", className = "border me-1"),
    dbc_badge("Dark", color = "white", text_color = "dark", className = "border me-1"),
    dbc_badge("Black", color = "white", text_color = "black", className = "border me-1"),
    dbc_badge("Muted", color = "white", text_color = "muted", className = "border me-1"),
    dbc_badge(
        "Light",
        color = "secondary",
        text_color = "light",
        className = "border me-1",
    ),
    dbc_badge("White", color = "secondary", text_color = "white", className = "border"),
])
